.github/workflows/reposchutz.yml

```diff
@@ -0,0 +1,40 @@
+name: repository
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, labeled, unlabeled]
+
+jobs:
+  check:
+    name: Protection checks
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
```

Comment:
There is a little too much going on here for this timeout -- let's make it 5 mins?

Reply:
Sure